prometric-derive/src/expand.rs

```diff
@@ -212,19 +316,29 @@ impl MetricBuilder {
         let help = &self.help;
         let mut doc_builder = format!(
             "{help}\n\
-            * Metric type: [prometric::{}]",
+            * Metric type: [::prometric::{}]",
```

Comment:
Ah nice, this means users don't have to explicitly import the crate?

Reply:
Well this is just on the doc builder, so not _necessary_  (and if we want to add a link we still need to put in the graves), but in general when macroexpanding it's good practice to prefix the crate with `::` to avoid referencing modules by the same name